sale_commission_formula/models/sale_commission_mixin.py

```diff
@@ -0,0 +1,29 @@
+#  -*- coding: utf-8 -*-
+#  Copyright 2019 Simone Rubino - Agile Business Group
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+from odoo.tools.safe_eval import safe_eval
+
+
+class SaleCommissionMixin(models.AbstractModel):
+    _inherit = 'sale.commission.mixin'
+
+    @api.model
+    def _get_formula_input_dict(self):
+        return {'line': self.object_id,
```

Comment:
Why do you overwrite this?

Reply:
@pedrobaeza I was right now testing this locally, I have overridden `sale.commission.mixin` by mistake, the correct model to override is `sale.commission.line.mixin`, fixed in https://github.com/OCA/commission/compare/6e24d9bad74ee98e56cba5be6ee4a95f43865341..6e998cdfb1e42ba552e154bb4e92b6033376a808